src/components/learner-credit-management/BudgetDetailRedemptions.jsx

```diff
@@ -21,8 +21,13 @@ const BudgetDetailRedemptions = ({ enterpriseFeatures, enterpriseUUID }) => {
     <section>
       <h3 className="mb-3">Spent</h3>
       <p className="small mb-4">
-        Spent activity is driven by completed enrollments. Enrollment data is automatically updated every 12 hours.
-        Come back later to view more recent enrollments.
+        Spent activity is driven by completed enrollments.
+        {(enterpriseOfferId || (subsidyAccessPolicyId && !enterpriseFeatures.topDownAssignmentRealTimeLcm)) && (
+          <>
+            Enrollment data is automatically updated every 12 hours.
+            Come back later to view more recent enrollments.
+          </>
```

Comment:
[curious/nit] Wondering if its worth specifically defining a test or adding assertions to validate if its rendering as expected given the conditional rendering.

Reply:
Pushed up a commit with added test assertions.